pkg/operator/ceph/csi/predicate.go

```diff
@@ -76,6 +90,13 @@ func predicateController(ctx context.Context, c client.Client) predicate.Funcs {
 		},
 
 		DeleteFunc: func(e event.DeleteEvent) bool {
+			// if the operator configuration file is deleted we want to reconcile to apply the
+			// configuration based on environment variables present in the operator's pod spec
+			if cm, ok := e.Object.(*v1.ConfigMap); ok {
+				// We don't want to use cm.Generation here, it case the operator was stopped and the
```

Comment:
Do we need this comment about Generation? That doesn't seem to apply to delete anyway?

Reply:
Correct!